packages/template-chakra-storefront/src/pages/account/wishlist/partials/wishlist-primary-action.test.js

```diff
@@ -11,21 +11,33 @@ import {renderWithProviders} from '../../../../utils/test-utils'
 import WishlistPrimaryAction from './wishlist-primary-action'
 import {screen, waitFor, act} from '@testing-library/react'
 import PropTypes from 'prop-types'
-import {rest} from 'msw'
 import {basketWithProductSet} from '../../../product-detail/index.mock'
 import {mockProductBundle} from '../../../../../mocks/product-bundle'
+import {prependHandlersToServer} from '../../../../../jest-setup'
+import Toaster, {toaster} from '../../../../components/toaster'
 
 const MockedComponent = ({variant}) => {
     return (
         <ItemVariantProvider variant={variant}>
             <WishlistPrimaryAction />
+            <Toaster toaster={toaster} />
```

Comment:
?

Reply:
We are using toast in WishlistPrimaryAction. Adding toast allows us to test if an action can trigger the toast success on the UI

Reply:
Check out the tests in this files. The tests for this component expect to see Toast sucessful message on screen. We missed the Toast here, so most of them failed previously
```
await waitFor(() => {
        expect(screen.getByText(/2 items added to cart/i)).toBeInTheDocument()
    })
```